src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportResource.kt

```diff
@@ -159,7 +160,7 @@ class ReportResource(
     @ParameterObject
     @PageableDefault(page = 0, size = 20, sort = ["incidentDateAndTime"], direction = Sort.Direction.DESC)
     pageable: Pageable,
-  ): SimplePage<ReportDto> {
+  ): SimplePage<ReportBasic> {
```

Comment:
Do you think it's worth changing the name of the Resource method to `getBasicReports()`? (and others accordingly as well?) - may help when navigating/orientating these big resources files.

Reply:
Similarly, the method name in the Service is now a bit ambiguous: `getReports()` ...are we getting back the basic ones or the ones with details?

Reply:
Other methods in the service may do different things, so there may be a consistency problem which may be surprising in the future.

For example, `getReports()` returns `ReportBasic` but `getReportById()` returns `ReportWithDetails?`.

I don't know if you like the idea of having more explicit method names  🤔